src/FrameBuffer.cpp

```diff
@@ -418,6 +423,42 @@ CachedTexture * FrameBuffer::_getSubTexture(u32 _t)
 	return m_pSubTexture;
 }
 
+CachedTexture * FrameBuffer::_copyFrameBufferTexture()
+{
+	if (m_copied)
+		return m_pFrameBufferCopyTexture;
+
+	if (m_pFrameBufferCopyTexture == nullptr) {
```

Comment:
Texture initialization better move to separate method for readability.
_initCopyTexture().